<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QDialog" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTipDuration">
   <number>-7</number>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabRead">
      <attribute name="title">
       <string>Read</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="loReadTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCreate">
      <attribute name="title">
       <string>Create</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="loCreateTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabUpdate">
      <attribute name="title">
       <string>Update</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="loUpdateTab"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDelete">
      <attribute name="title">
       <string>Delete</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QVBoxLayout" name="loDeleteTab"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
